apps/fe-admin/src/pages/OrderPage/OrderPage.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { Table, Sort } from '@shopery/ui-shared';
+import styles from './OrderPage.module.scss';
+
+const OrderPage = () => {
+  return (
+    <section className={styles.orderPage}>
+      <h1>Orders</h1>
+      <Sort />
+      <Table
+        admin={true}
```

Comment:
You can use this syntax
```suggestion
        admin
```

but it is incorrect that your shared table has dependency with admin
if you need actions, rename it to props with actions